fdbserver/IKeyValueStore.h

```diff
@@ -20,6 +20,8 @@
 
 #ifndef FDBSERVER_IKEYVALUESTORE_H
 #define FDBSERVER_IKEYVALUESTORE_H
+#include "flow/FastRef.h"
```

Comment:
There is still a bit of clean up left to do (mostly related to things i added to help with tracing) but i'll leave it until we are in a good spot (review wise).